src/foam/nanos/cron/Cron.js

```diff
@@ -83,7 +86,28 @@ foam.CLASS({
       ],
       javaReturns: 'void',
       javaCode:
-`super.runScript(x);
+`DAO notification = (DAO) x.get("notificationDAO");
+            
+Notification cronStartNotify = new Notification();
+StringBuilder startMsg = new StringBuilder();
+startMsg.append("Cron job: ").append(this.getId()).append(" started.");
+if ( ! SafetyUtil.isEmpty(this.getDescription()) ) {
+  startMsg.append(" Cron job description: ").append(this.getDescription());
+}
+cronStartNotify.setBody(startMsg.toString());
+notification.put(cronStartNotify);
```

Comment:
Wondering ...
1) Is Cron the terminology for scheduling used in the app/UI?  If not would it be better to use the same terminology in for user facing notifications?  e.g: Scheduled Job #123 started
2) Can we shorten the msg?  Maybe no need to repeat the phrase 'Cron job description' everytime.  Maybe better to compose the msg more tersely.  e.g:
```Scheduled job #<job-id> '<job-description>' started```
or perhaps
```Scheduled job `<job-description>` (id#<job-id>) ended```

Reply:
For #2 .. see @jlhughes suggestion earlier

Reply:
Crons will be administered by 'admin's of the system, not regular users.